charts/tfy-gpu-operator/templates/civo-talos-fake-toolkit-ready.yaml

```diff
@@ -0,0 +1,73 @@
+{{- if .Values.clusterType.civoTalos }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ .Release.Name }}-fake-toolkit-ready
+  namespace: tfy-gpu-operator
+spec:
+  selector:
+    matchLabels:
+      k8s-app: {{ .Release.Name }}-fake-toolkit-ready
+  template:
+    metadata:
+      labels:
+        name: {{ .Release.Name }}-fake-toolkit-ready
+        k8s-app: {{ .Release.Name }}-fake-toolkit-ready
+    spec:
+      volumes:
+        - name: run-nvidia
+          hostPath:
+            path: /run/nvidia/validations/
+      containers:
+        - name: main
+          image: alpine:3.19
+          command:
+            - sh
+            - '-c'
+          args:
+            - |-
+              set -ex;
+              touch /run/nvidia/validations/host-driver-ready;
+              touch /run/nvidia/validations/toolkit-ready;
```

Comment:
This is needed till gpu-operator adds support to disable validations via init containers